src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -555,7 +562,15 @@ internal static SaferPolicy GetSaferPolicy(string path, SafeHandle handle)
             }
             else
             {
-                throw new System.ComponentModel.Win32Exception(Marshal.GetLastWin32Error());
+                int lastError = Marshal.GetLastWin32Error();
+                if (lastError == NativeConstants.FUNCTION_NOT_SUPPORTED)
+                {
+                    _saferApiSupported = false;
```

Comment:
The var name "_saferApiSupported" is kind of misleading because it is broad in a sense that there are other native API calls (Cert-related) in this file that neither utilize this 'global flag' nor set it in case of similar error handling (Win32Exception(Marshal.GetLastWin32Error())).
So for consistency either this"_saferApiSupported" flag should be added to them; or simply renamed to something more narrow like "_SaferIdentifyLevelApiSupported".